good-citizen.md

```diff
@@ -0,0 +1,50 @@
+# Being a Good Citizen of Exercism 
+
+We want Exercism to be a great place to spend time, whoever you are.
+To ensure that Exercism is that, we place a strong focus on being good community citizens.
+This is formalised in our [Code of Conduct](https://exercism.io/code-of-conduct) but can be summed up simply in one sentence: **Be kind.**
+
+## The words we use
+
+There are lots of ways to embrace being kind within Exercism but the most powerful is to consider how your words affect others.
+
+Words in themselves are meaningless. 
+They are vibrations caused by puffs of air coming out of our lungs.
+However, because of **context**, words are powerful.
+
+It's easy to know how you intend your words to come across, but it's difficult to know how others hear your words.
+What we say is strongly tempered by our experiences in life, our cultural norms, and our primary spoken language.
+At Exercism, we believe it's better to choose words that are **clear** and **unambiguous** across cultures and, where possible, across spoken languages.
+This means being careful when using metaphors, acronyms or phrases that might only be understandable to someone from a specific background or with specific knowledge.
+It also means being open to learning about how people from different cultures might misunderstand your intentions, and changing the words you use accordingly.
+
+One example is that within Exercism we ask people not to use the word "guys" unless explicitely addressing a group of men. 
```

Comment:
As I said on Slack, I'd argue it's a bad word to use even if you address a group of men for the exact same reasons you outline below. It's more clear to use a word that's gendered everywhere, and not just in certain variations of English.